lighthouse-core/lib/network-recorder.js

```diff
@@ -27,13 +24,11 @@ class NetworkRecorder extends EventEmitter {
   }
 
   /**
-   * @return {Array<LH.Artifacts.NetworkRequest>}
+   * Returns the array of raw network request data without finalizing the initiator and
+   * redirect chain.
+   * @return {Array<NetworkRequest>}
    */
-  getInflightRecords() {
-    return this._records.filter(record => !record.finished);
-  }
-
-  getRecords() {
+  getRawRecords() {
```

Comment:
this rename felt necessary to differentiate from the records that have initiator and redirects set when invoked from `recordsFromLogs` I started to fix that discrepancy but this PR was big enough as-is